src/auth/authRepository.ts

```diff
@@ -34,63 +45,93 @@ export class AuthRepository implements IAuthRepository {
               `,
       [userName, userSurname, userEmail, userPassword]
     );
-
-    const notification = await this.database.query(`
-                insert into notifications ( user_id, code, type_of_notice)
-                VALUES ($1, $2, 'EMAIL_VERIFICATION');
-                `, [user.rows[0].id, code ]);
+    const expires_at = new Date(Date.now() + 15 * 60 * 1000);
+    await this.database.query(
```

Comment:
move logic with notifications folder
it will have NotificactionService that has NotificationRepository classes
NotificactionService must work with its repository and its responsibility to generate codes, so move function of generating code here

Reply:
also move creation and selecting of notification from authRepository to notifiactionReposiotry